src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String OPERATION_SUPPLY = "supply";
+
+    private static final String OPERATION_BUY = "buy";
+
+    private static final String OPERATION_RESULT = "result";
+
+    private static final String COMA_SEPARATOR = ",";
+
+    private static final String TEXT_OF_EXCEPTION = "File not found";
+
+    private static final String REGEX = "\\W";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supplyAmount = 0;
+        int buyAmount = 0;
+        StringBuilder builder = new StringBuilder();
```

Comment:
Let's try not to use StringBuilder as method parameter and returned result. Better to use String.